NuGet.Config

```diff
@@ -19,6 +19,7 @@
     <add key="dotnet.myget.org symreader" value="https://dotnet.myget.org/F/symreader/api/v3/index.json" />
     <add key="dotnet.myget.org symreader-portable" value="https://dotnet.myget.org/F/symreader-portable/api/v3/index.json" />
     <add key="dotnet.myget.org symreader-converter" value="https://dotnet.myget.org/F/symreader-converter/api/v3/index.json" />
+    <add key="dotnet.myget.org symreader-native" value="https://dotnet.myget.org/F/symreader-native/api/v3/index.json" />
```

Comment:
I feel like we have a feed per symreader assembly now.  This can't be helping our restore times.  Can we combine them into a single symreader feed?